### DISCUSSION TITLE
[Feature] Combined photo library

### DISCUSSION THREAD

**Discussion by figadore** [immich-app/immich#5692] [Feature Request]
### The feature

With the App-Which-Must-Not-Be-Named and Synology, there is an ability to have partner sharing enabled such that all photos show up combined with the other user(s). In Immich, so far all I've been able to do is go to each individual partner and see their specific photos. Behind the scenes they're all still stored in separate folders, so they can be filtered as needed, but especially when there is more than one partner it would be convenient to view the photos together.

One use case we have is a DSLR which we both use. For this, I'd probably create a separate user and upload this camera's photos as that user, and I'd like to see them combined with my phone's and my partner's phone's photos.

This is the last feature preventing me from ditching other photo apps, do you see this as useful functionality?

Bonus: One thing I like about Synology's implementation is having both a shared and a personal library, which means I can upload my screenshots and text message images to my personal library, and all of my camera's photos to the shared library.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by kaaku3** [comment]
They already of this feature, they call it partner sharing.
Currently you can't select what is shared, can only share the whole library and show as a combine timeline.
So your use case of creating a user just for sharing with both main accounts will work just fine.

After adding a partner the option is under account settings to show partner assets on the timeline.

You can also share individual albums, but I not sure if that currently appears on the timeline or not.... It didn't before.

**Comment by figadore** [comment]
> After adding a partner the option is under account settings to show partner assets on the timeline.

This is the setting I've been looking for. I didn't realize it had been added, but I just updated to v1.90.2 and now I see it. Thanks so much
